api/src/main/java/module-info.java

```diff
@@ -1109,6 +1111,37 @@
  * {@code @Transactional} is called in an environment where both Jakarta Transactions and
  * Jakarta CDI are available, the semantics of this annotation are observed during execution
  * of the repository operation.</p>
+ *
+ * <h2>Jakarta Concurrency</h2>
+ *
+ * <p>When Jakarta Concurrency is available, an asynchronous repository method may be
+ * annotated with {@code jakarta.enterprise.concurrent.Asynchronous} to cause the
+ * method to run asynchronously to the method invoker, as outlined by the section
+ * titled Asynchronous Methods in the Jakarta Concurrency specification.
+ * A {@code CompletionStage} return type must be parameterized with a type that
+ * would be a valid return type for a non-asynchronous repository method.</p>
+ *
+ * <p>For example, in the following example, the {@code setPriceAsync} method
+ * immediately returns a {@code CompletionStage<Integer>} to the caller.
+ * When the {@code UPDATE} finishes, the {@code CompletionStage} completes
```

Comment:
```suggestion
 * After the operation is performed by the database, the {@code CompletionStage} completes
```

Reply:
Not entirely sure of the best way to phrase this one, but "{@code UPDATE}" sounds too much like we're assuming this is SQL.